cfme/infrastructure/virtual_machines.py

```diff
@@ -904,6 +903,19 @@ def reconfigure(self, new_configuration=None, changes=None, cancel=False):
 
         # TODO This should (one day) return a VM reconfigure request obj that we can further use
 
+    @property
+    def cluster(self):
+        all_clusters = self.provider.get_clusters()
+        return next(cl for cl in all_clusters if cl.id == self.cluster_id)
+
+    @property
+    def host(self):
+        self.load_details()
+        host_name = InfoBlock.text('Relationship', 'Host')
```

Comment:
REQUIRED: please replace this with 
`
view = navigate_to(vm, 'Details)
view.entitiies.relatationship.get_text_of call
`